### PR TITLE
Add REST API to query Slobrok

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

File: orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java [hunk 1]
```diff
@@ -73,4 +76,32 @@ public InstanceStatusResponse getInstance(@PathParam("instanceId") String instan
...
+
+        List<Mirror.Entry> entries = slobrokMonitorManager.lookup(applicationId, pattern);
+        return entries.stream().map(SlobrokEntryResponse::fromMirrorEntry)
+                .collect(Collectors.toList());
+    }
+
+    private ApplicationInstanceReference parseInstanceId(@PathParam("instanceId") String instanceIdString) {
+        ApplicationInstanceReference instanceId;
+        try {
+            instanceId = parseAppInstanceReference(instanceIdString);
```
Comment (bjorncs) [inline orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java]: Consider removing `instanceId` variable and return result from `parseAppInstanceReference` directly.

File: orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java [hunk 2]
```diff
@@ -73,4 +76,32 @@ public InstanceStatusResponse getInstance(@PathParam("instanceId") String instan
...
+    @GET
+    @Path("/{instanceId}/slobrok")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<SlobrokEntryResponse> getSlobrokEntries(
+            @PathParam("instanceId") String instanceId,
+            @QueryParam("pattern") String pattern) {
+        ApplicationInstanceReference reference = parseInstanceId(instanceId);
+        ApplicationId applicationId = OrchestratorUtil.toApplicationId(reference);
+
+        if (pattern == null) {
```
Comment (bjorncs) [inline orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java]: Consider using `@DefaultValue` on query param.

**Review by bjorncs — APPROVED** [review]

File: orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java (hunk 2 above)
Comment (hakonhall) [reply orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java]: If you don't mind I'd keep this as-is, but I'll add a test that verifies null is handled to produce "**" pattern. The test wouldn't make sense with @DefaultValue.

File: orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java (hunk 1 above)
Comment (hakonhall) [reply orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java]: Done

**Comment by hakonhall** [comment]
Review comments handled in PR 3930
